ocw_oer_export/mapping_files/ocw_topic_to_oer_subject.csv

```diff
@@ -315,13 +315,13 @@ Society,Social Science,
 African-American Studies,Ethnic Studies,African-American Studies
 Asian Studies,World Cultures,Asian Studies
 European and Russian Studies,World Cultures,European and Russian Studies
-Gender Studies,Social Science|Women’s Studies,Gender Studies
+Gender Studies,Social Science|Gender and Sexuality Studies,Gender Studies
 Global Poverty,Cultural Geography,Global Poverty
 Indigenous Studies,Ethnic Studies|Cultural Geography,Indigenous Studies
 Latin and Caribbean Studies,World Cultures,Latin and Caribbean Studies
 Middle Eastern Studies,World Cultures,Middle Eastern Studies
 The Developing World,Cultural Geography,Developing World
-Women’s Studies,Gender and Sexuality Studies,Women's Studies
+Women’s Studies,Gender and Sexuality Studies,

```

Comment:
I think this is more correct: 

```suggestion
Women's Studies,Gender and Sexuality Studies,Gender and Sexuality Studies

```

1. Match the spelling in https://github.com/mitodl/ocw-hugo-projects/blob/27aafe988fc114c2c56a6ba226d12ca127695144/ocw-course-v2/ocw-studio.yaml#L829
2. Is there a reason why some mappings have keywords and others do not?

Reply:
@pdpinch 
> Is there a reason why some mappings have keywords and others do not?

They were provided that way in the file. However, I did notice a pattern (which is why I DM'd you about the keywords): We don't have `OER Subject` and `OER Keyword` mappings same in any case, except for the additions from this PR.

For example:
```
OCW Topic,OER Subject,OER Keyword
Business,Business and Communication,
```
I initially thought that if a keyword wasn’t present, the subject might officially serve as a keyword according to OER Commons. However, that is not currently present in the file. I also checked OER Commons but couldn’t find official mappings on their site.

I’ll proceed by adding the keywords like you said, but limited to the specific subjects in this PR/ticket. However, given this information, could you kindly confirm with Peter if any modifications to the mappings file are needed?

Here’s the file on the `main` branch for reference:
https://github.com/mitodl/ocw_oer_export/blob/main/ocw_oer_export/mapping_files/ocw_topic_to_oer_subject.csv

Reply:
Peter at OER Commons would like to have keywords for each course, but we don't have a way to provide those for most courses (only older legacy courses). 

> I’ll proceed by adding the keywords like you said, but limited to the specific subjects in this PR/ticket.

ok. I don't want to be making big changes here.